### PR TITLE
CMAX-283: Finish 6.2 backport

### PR DESCRIPTION
**Pull request by brandonmaynard**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/structure/page/_cq_infoProviders/.content.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (HitmanInWis) [inline ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/structure/page/_cq_infoProviders/.content.xml]: When we were debugging this, was it noticed that the asset share page resource type has a supertype of `core/wcm/components/page/v2/page`, which in turn has a supertype of `wcm/foundation/components/basicpage/v1/basicpage`, which doesnt exist on AEM 6.2?  I wonder if this is also why all the issues with the page properties not working.

Should we have instead updated the supertype for `core/wcm/components/page/v2/page` to `wcm/foundation/components/page` ?

**Review by HitmanInWis — CHANGES_REQUESTED** [review]

File: ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/structure/page/_cq_infoProviders/.content.xml (hunk 1 above)
Comment (brandonmaynard) [reply ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/structure/page/_cq_infoProviders/.content.xml]: That was exactly the issue. I added this node and then later changed the v2 definition to match the 6.2 OOTB `wcm/foundation/components/page` as it's supertype while trying to figure out the page dialog issues we were facing. I can test if removing this node will still work after making the supertype adjustment if you'd like

Comment (HitmanInWis) [reply ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/structure/page/_cq_infoProviders/.content.xml]: Yes, please give it a shot.

Comment (brandonmaynard) [reply ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/structure/page/_cq_infoProviders/.content.xml]: That worked. Removing this file

**Review by HitmanInWis — APPROVED** [review]
All comments in this PR as well as https://github.com/HS2-SOLUTIONS/asset-share-commons/pull/3 and https://github.com/HS2-SOLUTIONS/asset-share-commons/pull/4 are addressed.

**Pull request merged**